src/odemis/driver/smaract.py

```diff
@@ -4018,6 +4255,8 @@ def __init__(self):
             SA_PS_STABLE_STATE_CHANGED_EVENT: SA_PS_SYS_IS_STABLE_PROP,
                                   }
 
+        self.err_code = {}
+
```

Comment:
What is this for? I feel it's some kind of hack for the simulator to work. At least add a comment how this works, or fix the simulator to behave more like the normal library.